### ISSUE TITLE
[BUG] Does not regenerate docs on changes

### ISSUE THREAD

**Issue by pronebird** [JScearcy/rust-doc-viewer#56]
**Describe the bug**
Doc viewer shows stale docs after changing the doc comments in the source code.

**To Reproduce**
1. Add a piece of code and document it with `///`
2. Cmd+shift+p then select Rust doc: viewer
3. Go back to source code and remove the `///` comment made in the step 1
4. Cmd+shift+p then select Rust doc: viewer 

**Expected behavior**
Expected to see new docs but I see old ones.

**Screenshots**

**Additional context**

**Comment by JScearcy** [comment]
Yeah there's no fs watcher today, but this seems reasonable / straight forward to add, like previous issues I've missed notifications for this repo 🤔 so I'll pick this up in the coming weeks
